packages/flutter_test/lib/src/binding.dart

```diff
@@ -1339,39 +1362,21 @@ class LiveTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
   /// asynchronous pause in the test (as would normally happen when
```

Comment:
I think our API doc should start with a short paragraph with a sentence. The old doc does not seem to follow that convention. Maybe we shall fix it too. CC @Hixie @goderbauer

Reply:
Added a new sentence.

Reply:
Yes, if you have the chance please change the first paragraph to a one-sentence summary.

Reply:
I like the new sentence, but the sentence after that. Oy. 

Original sentence:
```
  /// Whether to have [pump] with a duration only pump a single frame
  /// (as would happen in a normal test environment using
  /// [AutomatedTestWidgetsFlutterBinding]), or whether to instead
  /// pump every frame that the system requests during any
  /// asynchronous pause in the test (as would normally happen when
  /// running an application with [WidgetsFlutterBinding]).
```

Clearer sentence:

```
  /// Whether [pump] (with a duration) pumps a single frame
  /// (as would happen in a normal test environment using
  /// [AutomatedTestWidgetsFlutterBinding]), or
  /// pumps every frame that the system requests during an
  /// asynchronous pause (as would normally happen when
  /// running an application with [WidgetsFlutterBinding]) ...
```

Then what? The original sentence needs a final clause. Whether it's one pump or many... what then?

Reply:
I changed the sentence to "The policy decides ..." to make it complete. I can't think of a clause.